packages/mobile/src/components/details-tile/DetailsTileHasAccess.tsx

```diff
@@ -30,22 +31,33 @@ const messages = {
   specialAccess: 'SPECIAL ACCESS',
   payToUnlock: 'Pay to Unlock',
   unlockedCollectibleGatedPrefix: 'A Collectible from ',
-  unlockedCollectibleGatedSuffix:
-    ' was found in a linked wallet. This track is now available.',
+  unlockedCollectibleGatedSuffix: (
+    contentType: PurchaseableContentType | 'item' = 'item'
+  ) => ` was found in a linked wallet. This ${contentType} is now available.`,
   ownerCollectibleGatedPrefix:
     'Users can unlock access by linking a wallet containing a collectible from ',
   unlockedFollowGatedPrefix: 'Thank you for following ',
-  unlockedFollowGatedSuffix: '! This track is now available.',
+  unlockedFollowGatedSuffix: (
```

Comment:
when are we using `item`? don't seem to see an instance

Reply:
We're hopefully not, I was just adding it as a backup in case an `undefined` somehow makes its way in

Reply:
`You've purchased this item` seems like a better fallback than `You've purchased this undefined` lol

Reply:
smart

Reply:
update: decided to remove. The types dont allow for undefined and this is set manually so it should never be the case